packages/modules/web_themes/koala/source/src/stores/mqtt-store-model.ts

```diff
@@ -127,7 +127,7 @@ export interface vehicleInfo {
   model: string;
 }
 
-export interface vehicleSocModule {
+export interface vehicleSocModuleConfig {
```

Comment:
```suggestion
export interface VehicleSocModuleConfig {
```
Bitte auch in Zeile 125 korrigieren.

Reply:
korrigiert